core/embed/trezorhal/unix/certs/gen.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+cd "$(dirname "$0")"
+for file in *.der
+do
+    header_file="${file%.der}.h"
+    xxd -i "$file" | sed 's/unsigned/static const unsigned/'> "$header_file"
```

Comment:
Suggestion: Add a comment at the top of the header file saying that it was autogenerated by gen.sh.

Reply:
https://github.com/trezor/trezor-firmware/commit/3959a6fdff11fb409b1f3914e09c173ff17ca6f2